spec/date.html

```diff
@@ -126,7 +126,9 @@ <h1>get Temporal.Date.prototype.year</h1>
       <emu-alg>
         1. Let _temporalDate_ be the *this* value.
         1. Perform ? RequireInternalSlot(_temporalDate_, [[InitializedTemporalDate]]).
-        1. Return _temporalDate_.[[ISOYear]].
+        1. Let _calendar_ be _temporalDate_.[[Calendar]].
+        1. Let _method_ be ? Get(_calendar_, *"year"*).
+        1. Return ? Call(_calendar_, _method_, « _temporalDate_ »).
```

Comment:
I think these are backwards, in https://tc39.es/ecma262/#sec-call the function object comes before the `this` value.